release/ray_release/alerts/handle.py

```diff
@@ -13,14 +13,24 @@
 
 
 result_to_handle_map = {
-    "default": default.handle_result,
-    "long_running_tests": long_running_tests.handle_result,
-    "rllib_tests": rllib_tests.handle_result,
-    "tune_tests": tune_tests.handle_result,
-    "xgboost_tests": xgboost_tests.handle_result,
+    "default": (default.handle_result, default.REQ_NON_EMPTY_RESULT),
+    "long_running_tests": (
+        long_running_tests.handle_result,
+        long_running_tests.REQ_NON_EMPTY_RESULT,
+    ),
+    "rllib_tests": (rllib_tests.handle_result, rllib_tests.REQ_NON_EMPTY_RESULT),
+    "tune_tests": (tune_tests.handle_result, tune_tests.REQ_NON_EMPTY_RESULT),
+    "xgboost_tests": (xgboost_tests.handle_result, xgboost_tests.REQ_NON_EMPTY_RESULT),
 }
 
 
+def require_non_empty_result(test: Test) -> bool:
```

Comment:
Nit: It's actually any result, not just empty

```suggestion
def require_result(test: Test) -> bool:
```